builds/test_python_runtime.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ARCHIVE_FILEPATH="${1:?"Error: The filepath of the Python runtime archive must be specified as the first argument."}"
+
+# We intentionally extract the Python runtime into a different directory to the one into which it
+# was originally installed before being packaged, to check that relocation works (since buildpacks
+# depend on it). Since the Python binary was built in shared mode, `LD_LIBRARY_PATH` must be set
+# when relocating, so the Python binary (which itself contains very little) can find `libpython`.
+INSTALL_DIR=$(mktemp -d)
+PYTHON="${INSTALL_DIR}/bin/python"
+export LD_LIBRARY_PATH="${INSTALL_DIR}/lib/"
+
+tar --zstd --extract --verbose --file "${ARCHIVE_FILEPATH}" --directory "${INSTALL_DIR}"
+
+# Check Python exists at the `python` alias (and not just `python3`) and can run.
+"${PYTHON}" --version
+
+# Check that all dynamically linked libraries exist in the run image (since it has fewer packages than the build image).
+if find "${INSTALL_DIR}" -name '*.so' -exec ldd '{}' + | grep 'not found'; then
+  echo "The above dynamically linked libraries were not found!"
+  exit 1
+fi
```

Comment:
[Talking out loud] This is neat. You're iterating through all shared object files in the python directory and making sure they're findable via `ldd`. Smart, and produces a good error message stating what expected package was missing. I like it.

Reply:
Yeah this is borrowed from:
https://github.com/heroku/buildpacks-python/blob/de378063c83e1eec6302e79f93f189162b5bbbbf/tests/python_version_test.rs#L121-L123